bootstrap/helpers/parsers.php

```
@@ -1445,6 +1446,23 @@ function applicationParser(Application $resource, int $pull_request_id = 0, ?int
         $parsedServices->put($serviceName, $payload);
+        if (isset($isDatabase) && $isDatabase) {
+            $detectedDatabases->put($serviceName, $service);
+        }
+    }
+    if ($isPullRequest) {
+        $resourceToSync = $resource->previews()->where('pull_request_id', $pullRequestId)->first();
+        if (! $resourceToSync) {
+            \Illuminate\Support\Facades\Log::warning('Preview not found for application parsing', [
+                'application_id' => $resource->id,
+                'pull_request_id' => $pullRequestId,
+            ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'applicationParser'. (undefined)

(StaticAccess)